tests/functional/v2/server/testGetStorage.js

```diff
@@ -0,0 +1,113 @@
+const assert = require('assert');
+const async = require('async');
+const uuid = require('uuid');
+
+const UtapiClient = require('../../../../libV2/client');
+const { client: warp10 } = require('../../../../libV2/warp10');
+const config = require('../../../../libV2/config');
+const { CacheClient, backends: cacheBackends } = require('../../../../libV2/cache');
+const { IngestShard } = require('../../../../libV2/tasks');
+const { now } = require('../../../../libV2/utils');
+const { generateCustomEvents } = require('../../../utils/v2Data');
+
+const getClient = () => new CacheClient({
+    cacheBackend: new cacheBackends.RedisCache(
+        config.cache,
+    ),
+    counterBackend: new cacheBackends.RedisCache(
+        config.cache,
+    ),
+});
+
+// eslint-disable-next-line func-names
+describe('Test getStorage handler', function () {
+    this.timeout(120000);
+    let client;
+
+    let events;
+    let totals;
+
+    let cacheClient;
+    let ingestTask;
+
+    beforeEach(async () => {
+        client = new UtapiClient({
+            drainDelay: 5000,
+        });
+
+        const { events: _events, totals: _totals } = generateCustomEvents(
+            now() - (120 * 1000000), now() - (30 * 1000000), 50, {
+                [uuid.v4()]: { [uuid.v4()]: [uuid.v4()] },
+            },
+        );
+        events = _events;
+        totals = _totals;
+
+        cacheClient = getClient();
+        await cacheClient.connect();
+
+        ingestTask = new IngestShard();
+        ingestTask._program = { lag: 0 };
+        await ingestTask._cache.connect();
+    });
+
+    it('should get the current storage for an account with a empty cache', async () => {
+        await warp10.ingest({ className: 'utapi.event' }, events);
+
+        await async.eachOf(totals.accounts, async (total, acc) => {
```

Comment:
what's inside `acc`?
I am guessing in Vault I have to use getStorage and `acc`

Reply:
`acc` is the accounts canonical Id